cookiecutter.json

```diff
@@ -5,12 +5,12 @@
     "author": "Jane Doe",
     "author_email": "jane_doe@imperial.ac.uk",
     "rse_team_as_coauthor": false,
-    "use_bsd3_licence": false,
+    "use_bsd3_license": true,
     "packaging": [
         "poetry",
         "pip-tools"
     ],
-    "mkdocs": false,
+    "mkdocs": true,
```

Comment:
Are we sure we want docs by default? For random researcher projects, they might not want a GitHub Pages site. *We* probably always want docs though.

I don't think this one matters as much, so I think it's fine either way.

Reply:
I think this is fine because it sets up all the mkdocs stuff to build the docs, but doesn't actually provide any of the GitHub-related infrastructure to deploy them. So nobody is going to accidentally have a pages site. This was also originally done so I could check it in CI, but I think it makes sense as a default.